### PR TITLE
Update global footer to use IO logo, and match main content width

### PR DESCRIPTION
**Pull request by nr-kkenney**
## Summary

* added in placeholder logo which is selected when the page url contains
'instant-observability'. once we have the real logo, we only need to
swap it with the placeholder.
* added in css styling to have the footer match the main content width

## Screenshots
Screen Shot 2021-09-15 at 9 09 23 AM
Screen Shot 2021-09-15 at 9 09 32 AM
Screen Shot 2021-09-15 at 9 09 44 AM

Resolves: https://github.com/newrelic/gatsby-theme-newrelic/issues/444

### CODE REVIEW

File: src/@newrelic/gatsby-theme-newrelic/components/Logo.js [hunk 1]
```diff
@@ -2,38 +2,113 @@ import React from 'react';
...
+        <path d="M86.25,40.29c0,1,.47,1.51,1.41,1.51h2.65v5.62H87.66c-.94,0-1.41.5-1.41,1.51V66.87a4,4,0,0,1-1.12,2.91,3.84,3.84,0,0,1-2.88,1.14h-5V67.18h3.59q1.41,0,1.41-1.56V49a4,4,0,0,1,1.11-2.91,3.92,3.92,0,0,1,2.94-1.15h.63V44.3h-.63a3.92,3.92,0,0,1-2.94-1.15,4,4,0,0,1-1.11-2.91V23.6Q82.2,22,80.79,22H77.2V18.3h5a3.84,3.84,0,0,1,2.88,1.14,4,4,0,0,1,1.12,2.91Z" />
+        <rect fill="#0ab0bf" x="31.42" y="32.05" width="32.22" height="5" />
+        <rect fill="#fda032" x="39.27" y="42.05" width="15.3" height="5" />
+        <rect fill="#66e6a8" x="58.76" y="42.05" width="15.3" height="5" />
+        <rect fill="#70ccd3" x="39.27" y="52.05" width="20.38" height="5" />
+      </g>
+    </svg>
+  );
+
+  /* TODO: replace with official logo once ready. for now, using new relic default logo. */
```
Comment (LizBaker) [inline src/@newrelic/gatsby-theme-newrelic/components/Logo.js:39]:
```suggestion
```

Comment (nr-kkenney) [reply src/@newrelic/gatsby-theme-newrelic/components/Logo.js:39]: 🙈

**Comment by LizBaker** [comment]
couple comments:

1. sorry if i missed it discussed elsewhere 😭 but why aren't we making the footer full width on the main page?
2. in the details page the footer is almost full width except a little left margin which feels off to me, especially with the bread crumbs bar being full width
3. clicking the logo takes me back to the developer homepage but should probably link back to the IO page

**Review by LizBaker — APPROVED** [review]

**Comment by nr-kkenney** [comment]
1, 2. made changes for the footer length / placement. now it matches the length of the main content container on both the IO page and quickstart details page.
3. i removed that AC and kind of explained on the ticket. the footer doesn't have a way to change what it links to currently, so we would have to add an additional prop to the component in the theme for this. but zack and clinton had questioned if that was worth it since this is not really functionality we have and exists only because we have the two sites combined into one for the moment.

**Pull request merged**